Added animate function

genliang opened the pull request:
I tried to make changes to terrain animate function to use it in skybox. But the color does change from day to night. It just shows night color. Is there another function that can make the sky change color for each for loop?

genliang committed:
First observation

looking out for the animate function and placing them into the light
object. Turns out to have error due to undeclared variables in the
animate function.

genliang committed:
Added animate function in skybox

Added animate function in skybox. A for loop is added, trying to
accomplish the animation from day to night

wizztjh commented:
but you want to make the sun set right? So, you will need to update the inclination of the sunSphere.

wizztjh commented:
you can use Math.sin(elapsedTime) to bounce the value from 0 to1 then 1 to 0

genliang committed:
change position of sun

position of sun changes, but still no animation from original position
to lower position

genliang commented:
I tried to use the Math.sin(elapsedTime) but it didn't work. Now when the inclination is changed, the sunSphere changes position. However it still could not animate. I tried to add sunSphere into the animate function at the bottom. Still does not work.
